apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -263,6 +264,13 @@ export const DEFAULT_BLOCKS: Record<
       },
     ],
   },
+  [SEARCHABLE_TABLE_TYPE]: {
+    type: `${SEARCHABLE_TABLE_TYPE}`,
+    dataSource: {
+      type: "dgs",
+      resourceId: "d_3c55210de27fcccda2ed0c63fdd2b352", // hardcoded
+    },
+  },
```

Comment:
adding for type completeness: otherwise not useded as it cannot be added as a component